src/export/ExtensionExporter.ts

```diff
@@ -1,15 +1,27 @@
 import { StructureDefinitionExporter } from './StructureDefinitionExporter';
 import { FSHTank } from '../import/FSHTank';
 import { StructureDefinition } from '../fhirtypes';
+import { FHIRDefinitions } from '../fhirdefs';
 
 export class ExtensionExporter extends StructureDefinitionExporter {
-  constructor() {
-    super();
+  constructor(FHIRDefs: FHIRDefinitions) {
+    super(FHIRDefs);
   }
 
-  // eslint-disable-next-line
+  /**
+   * Exports Extensions to StructureDefinitions
+   * @param {FSHTank} tank - The FSH tank we are exporting
+   * @returns {StructureDefinition[]}
+   */
   export(tank: FSHTank): StructureDefinition[] {
-    // TODO: Make meaningful export of StructureDefinitions from contents of tank
-    return [];
+    const structDefs: StructureDefinition[] = [];
+    for (const doc of tank.docs) {
+      for (const [, extension] of doc.extensions) {
```

Comment:
Honest question... do you think that it might be more straightforward to iterate over `doc.extensions.values()`?
```ts
for (const extension of doc.extensions.values()) {
```

Reply:
Aw no one liked my cheeky syntax